types/node/v8/index.d.ts

```diff
@@ -5575,6 +5576,48 @@ declare module "util" {
     export namespace promisify {
         const custom: symbol;
     }
+
+    export interface TextDecodeOptions {
+        fatal?: boolean;
+        ignoreBOM?: boolean;
+    }
+    export interface TextDecoderOptions {
+        fatal?: boolean;
+        ignoreBOM?: boolean;
+    }
+    interface TextDecoder {
+        readonly encoding: string;
+        readonly fatal: boolean;
+        readonly ignoreBOM: boolean;
+        decode(
+            input?:
+            | Int8Array
+            | Int16Array
+            | Int32Array
+            | Uint8Array
+            | Uint16Array
+            | Uint32Array
+            | Uint8ClampedArray
+            | Float32Array
+            | Float64Array
+            | DataView
+            | ArrayBuffer
+            | null,
+            options?: TextDecodeOptions
+        ): string;
+    }
+    export var TextDecoder: {
+        prototype: TextDecoder;
+        new (encoding?: string, options?: TextDecoderOptions): TextDecoder;
+    };
+    interface TextEncoder {
+        readonly encoding: string;
+        encode(input?: string): Uint8Array;
+    }
+    export var TextEncoder: {
```

Comment:
Similar as above I think a class would be a better match then `interface` + `var`,